### DISCUSSION TITLE
Calculations

### DISCUSSION THREAD

**Discussion by groiso** [YetiForceCompany/YetiForceCRM#15849] [General] [answered]
Hi,

I need to create a field and do calculations in it, where I can create formulas like: (10 - (2+2))/2
Thanks

**Comment by s2b-git** [comment, marked as answer]
> workflows

**Reply by groiso** [reply]
But in workflows, the result of a calculation only appears after saving the records and not while inserting the data.

**Reply by mariuszkrzaczkowski** [reply]
what question such an answer, write a more specific question will be a better answer

**Reply by groiso** [reply]
As far as I know, workflows only update fields after saving.
What is required for the total field to be updated when I enter values in two fields, in addition to workflows?

**Reply by rskrzypczak** [reply]
CRM does not provide such functionalities.
